src/main/java/uk/gov/justice/digital/hmpps/csr/api/controllers/DetailController.kt

```diff
@@ -19,16 +22,20 @@ import uk.gov.justice.digital.hmpps.csr.api.dto.DetailDto
 import uk.gov.justice.digital.hmpps.csr.api.service.DetailService
 import java.time.LocalDate
 
-@Tag(
-  name = "details",
-  description = "Details of shifts from CSR. NOTE: All of these endpoints use the header value 'X-Region' to select the required region"
-)
+@Tag(name = "details", description = "Details of shifts from CSR")
 @RestController
 @RequestMapping(produces = [APPLICATION_JSON_VALUE])
 class DetailController(private val detailService: DetailService) {
 
   @Operation(summary = "Retrieve all details for a user between two dates")
+  @Parameter(
+    `in` = ParameterIn.HEADER,
+    name = "X-Region",
+    description = "the number of the required region, 1-6",
+    required = true
+  )
   @GetMapping("/user/details")
```

Comment:
why was this needed?

Reply:
its not actually. I originally wanted to add the region description to highlight that it is needed for the manual admin endpoints too, but now Ive changed the region parameter to be on path. I guess its harmless tho